sdks/python/apache_beam/runners/worker/opcounters_test.py

```diff
@@ -41,6 +47,55 @@ def __init__(self):
     pass
 
 
+class TransformIoCounterTest(unittest.TestCase):
+
+  def setUp(self):
+    if not statesampler.FAST_SAMPLER:
+      raise SkipTest('State sampler not compiled.')
```

Comment:
<!--thread_id:cc_163698968_t; commit:b68f9f26428de8ed2f34c0919ee91a1662402d86; resolved:1-->
<!--section:context-quote-->
> **aaltay** wrote:
> Have you tried using unittest.skipTest() (https://docs.python.org/2/library/unittest.html#unittest.TestCase.skipTest). We are using that in a few other places.

<!--section:body-->
With the new pure python state sampler implementation, skipping is no longer necessary.